.github/CODEOWNERS

```
@@ -1,4 +1,6 @@
 /packages/@expo/cli                             @EvanBacon @bycedric
+/packages/@expo/config-plugins                  @EvanBacon @douglowder
+/packages/@expo/config-types                    @EvanBacon @douglowder
```

```suggestion
/packages/@expo/config                                @EvanBacon
/packages/@expo/prebuild-config                @EvanBacon
/packages/@expo/config-plugins                  @EvanBacon @brentvatne
/packages/@expo/config-types                    @EvanBacon @douglowder
```

Given the long history of the long, delicate history of the config packages, would be good if nothing merges without the authors giving it a once-over. config-types should be fine since @douglowder got the crash course on how it works this afternoon.